persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -76,6 +76,96 @@ UPDATE Conversation
 SET last_read_date = :last_read_date
 WHERE qualified_id = :qualified_id;
 
+CREATE VIEW IF NOT EXISTS ConversatonsList AS
+SELECT
+Conversation.qualified_id AS qualifiedId,
+CASE (Conversation.type)
+    WHEN "ONE_ON_ONE" THEN  User.name
+    WHEN "CONNECTION_PENDING" THEN connection_user.name
+    ELSE Conversation.name
+    END AS name,
+Conversation.type,
+Call.status AS callStatus,
+CASE (Conversation.type)
+    WHEN "ONE_ON_ONE" THEN  User.preview_asset_id
+    WHEN "CONNECTION_PENDING" THEN connection_user.preview_asset_id
+    END AS previewAssetId,
+Conversation.muted_status AS mutedStatus,
+CASE (Conversation.type)
+    WHEN "ONE_ON_ONE" THEN User.team
+    ELSE Conversation.team_id
+    END AS teamId,
+Conversation.last_modified_date AS lastModifiedDate,
+Conversation.last_read_date AS lastReadDate,
+CASE (Conversation.type)
+    WHEN "ONE_ON_ONE" THEN  User.user_availability_status
+    WHEN "CONNECTION_PENDING" THEN connection_user.user_availability_status
+    END AS userAvailabilityStatus,
+CASE (Conversation.type)
+    WHEN "ONE_ON_ONE" THEN  User.user_type
+    WHEN "CONNECTION_PENDING" THEN connection_user.user_type
+    END AS userType,
+CASE (Conversation.type)
+    WHEN "ONE_ON_ONE" THEN  User.bot_service
+    WHEN "CONNECTION_PENDING" THEN connection_user.bot_service
+    END AS botService,
+CASE (Conversation.type)
+    WHEN "ONE_ON_ONE" THEN  User.deleted
+    WHEN "CONNECTION_PENDING" THEN connection_user.deleted
+    END AS userDeleted,
+CASE (Conversation.type)
+    WHEN "ONE_ON_ONE" THEN  User.connection_status
+    WHEN "CONNECTION_PENDING" THEN connection_user.connection_status
+    END AS connectionStatus,
+CASE (Conversation.type)
+    WHEN "ONE_ON_ONE" THEN  User.qualified_id
+    WHEN "CONNECTION_PENDING" THEN connection_user.qualified_id
+    END AS otherUserId,
```

Comment:
We are pulling almost all fields from the `connection_user` here why not pull in the whole `connection_user`  instead? It would then be nicely typed.